envoy_iptables/Makefile

```diff
@@ -9,9 +9,8 @@ build: image
 
 .PHONY: image
 image:
-	docker build -t $(REPOSITORY):latest -t $(REPOSITORY):$(VERSION) .
+	docker buildx build --platform=linux/amd64,linux/arm64 -t $(REPOSITORY):latest -t $(REPOSITORY):$(VERSION) .
 
 .PHONY: push
 push: build
```

Comment:
True, but since `docker buildx build` should cache the intermediate results, running it twice is a no-op anyway. I can remove the dep there though, since it should if nothing else save a few seconds.